packages/dapi/Dockerfile

```diff
@@ -27,6 +27,7 @@ COPY packages/dapi-grpc packages/dapi-grpc
 COPY packages/js-dpp packages/js-dpp
 COPY packages/js-grpc-common packages/js-grpc-common
 COPY packages/feature-flags-contract packages/feature-flags-contract
+COPY packages/masternode-reward-shares-contract packages/masternode-reward-shares-contract
```

Comment:
Does DAPI uses this contract as a production dependency

Reply:
It uses DPP which uses this dependency
Otherwise it fails with the error

Reply:
I got it! Thank you